packages/block-library/src/social-link/index.js

```diff
@@ -17,5 +16,4 @@ export { metadata, name };
 export const settings = {
 	icon,
 	edit,
-	variations,
```

Comment:
We'll need to add back fallback support for WP < 5.8, but leaving this as is while I develop